### DISCUSSION TITLE
Welcome to termux-desktop-lxqt Discussions!

### DISCUSSION THREAD

**Discussion by JesusChapman** [Yisus7u7/termux-desktop-lxqt#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JesusChapman** [comment]
@suhan-paradkar
@librehat 
What they think about  the new look I gave to lxqt, it took me days of practice, at last achieving a harmony between the qt5 and gtk themes.

**Comment by LeonardoE549** [comment]
Hola, seguí  los pasos que me indicaba la instalacion y no tuve ningun problema . Pero al abrir el vnc solo me aparece la pantalla en negro 😥

**Reply by JesusChapman** [reply]
Add in: (~/.vnc/xstartup)

```bash
#!/data/data/com.termux/files/usr/bin/sh
lxqt-session &
```

And chmod +x ~/.vnc/xstartup
